src/components/PagePagination/Pagination.js

```diff
@@ -0,0 +1,71 @@
+import classnames from 'classnames';
+import { usePagination, DOTS } from './PagePagination';
+import './PagePagination.scss';
+
+export const Pagination = totalBookCount => {
+   const {
+    onPageChange,
+    totalCount,
+    siblingCount = 1,
+    currentPage,
+    pageSize,
+    className
+   } = totalBookCount;
+
+   const paginationRange = usePagination({
+    currentPage,
+    totalCount,
+    siblingCount,
+    pageSize
+   });
+
+   if (currentPage === 0 || paginationRange.length < 2){
+    return null;
+   }
+
+   const onNext = () => {
+    onPageChange(currentPage + 1);
+   }
+
+   const onPrevious = () =>{
+    onPageChange(currentPage - 1);
+   }
+
+   let lastPage = paginationRange[paginationRange.length - 1];
+   return (
+    
+    <ul className={classnames('pagination-container', {[className]: className})}>
+       
+        <li className={classnames('pagination-item', {
+            disabled: currentPage === 1
+        })}
+        onClick={onPrevious}
+        >
+            <div className="arrow left"/>
+        </li>
+        {paginationRange.map(pageNumber => {
+         if (pageNumber === DOTS){
+            return<li className="pagination-item dots">&#8230;</li>
+         }   
+         return(
+            <li className={classnames("pagination-item", {
+                selected: pageNumber === currentPage
+            })}
+            onClick={() => onPageChange(pageNumber)}
+            >
+                {pageNumber}
+            </li>
+         )
+        })}
+        <li
+        className={classnames("pagination-item", {
+            disabled: currentPage === lastPage
+        })}
+        onClick={onNext}
+        >
+            <div className="arrow right"/>
+        </li>
+    </ul>   
```

Comment:
Before making the page options, render out bookList. For each item in bookList we can render out a BookItem.